### PR TITLE
Please add QQCS to the list of participating projects.

### PR DESCRIPTION
**Pull request by dde**

### CODE REVIEW

File: participating-projects.md [hunk 1]
```diff
@@ -262,4 +262,32 @@ The Quantum Open Source Foundation (QOSF) Monthly Challenges aim to help partici
...
+
+It is an educational tool that can be easily used by students new to Quantum Computing.
+
+It provides automatic mathematical analysis of circuits by incorporating the matrix mathematics necessary to provide insight to circuit operation, and by displaying the details at each execution step, something not available from quantum computer simulated execution.
+
+It can be executed in interactive or batch mode.
+
+For example, here is the QQCS statement for the classic superdense coding circuit of Eve, Alice, and Bob.
+
+```|00>:H_:C01:X_:Z_:C01:H_:MM```
```
Comment (crazy4pi314) [inline participating-projects.md]:
```suggestion

```

Comment (crazy4pi314) [reply participating-projects.md]: I would suggest for brevity the samples/further detail be omitted.

**Review by crazy4pi314 — CHANGES_REQUESTED** [review]
LGTM, just a suggestion to shorten a bit to be more uniform with the other listings.

**Comment by dde** [comment]
Modified as requested.

**Comment by crazy4pi314** [comment]
LGTM! thanks🙏

**Pull request merged**